monai/data/test_time_augmentation.py

```
@@ -152,20 +158,21 @@ def __init__(
+
+        for idx, t in enumerate(transforms):
+            if isinstance(t, Randomizable):
+                randoms.append(t)
+                if self.apply_inverse_to_pred and not isinstance(t, InvertibleTransform):
+                    warns.append(f"Transform #{idx} (type {type(t).__name__}) is random but not invertible.")
+
+        if len(randoms) == 0:
+            warns.append("TTA usually requires at least one `Randomizable` transform in the given transform sequence.")
+
+        if len(warns) > 0:
+            warnings.warn("TTA has encountered issues with the given transforms:" + "\n  ".join(warns))
```

Ruff (0.14.13)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)